packages/dso-toolkit/src/components/card/card.scss

```diff
@@ -12,32 +12,34 @@
 }
 
 dso-card {
-  &[is-selectable],
-  &[has-image] {
-    @extend %card-grid;
-  }
-
-  a[slot="heading"] {
+  .dso-card-heading a {
```

Comment:
Iets in mij zegt dat deze styling niets doet

Reply:
Ik denk dat deze stylesheet nog eens tegen het licht gehouden moet worden: Welke styling hoort er bij de HTML/CSS implementatie, en welke styling gaat er over de kindjes die aan de HTML/CSS en Web Component implementatie worden gegeven

Reply:
Pak jij deze, @hansgrimm ?

Reply:
> Iets in mij zegt dat deze styling niets doet

tot op zekere hoogte. wel wat uitgeschoond.

Reply:
Ok. Ik ga kijken